stdlib/2and3/logging/handlers.pyi

```diff
@@ -40,6 +41,7 @@ if sys.version_info >= (3,):
 
 if sys.version_info >= (3,):
     class RotatingFileHandler(BaseRotatingHandler):
+        terminator = ...  # type: str
```

Comment:
This is redundant with BaseRotatingHandler, is it not?

Reply:
Ping, could you remove this?

Reply:
Done, sorry for the delay.